base/Dockerfile

```diff
@@ -36,7 +36,8 @@ RUN curl -o ruby.tgz \
 ENV GEM_SPEC_CACHE "/tmp/gemspec"
 RUN echo 'gem: --no-document' > $HOME/.gemrc
 RUN gem update --system ${RUBYGEMS_VERSION}
-RUN gem install -v ${BUNDLER_VERSION} bundler
+# we have to force install bundler as it is a vendored gem since rubygems 2.7.0
```

Comment:
Can't we just rely on the vendored bundler then?

Reply:
I'm a fan of using latest stable. Do you see any downsides?

Reply:
No, I'm fine keeping it.